src/main/java/seedu/address/ui/ElderlyListPanel.java

```diff
@@ -27,12 +28,13 @@ public ElderlyListPanel(ObservableList<Elderly> elderlyList) {
         super(FXML);
         elderlyListView.setItems(elderlyList);
         elderlyListView.setCellFactory(listView -> new ElderlyListViewCell());
+        ObservableList<Node> children = elderlyListView.getChildrenUnmodifiable();
     }
 
     /**
      * Custom {@code ListCell} that displays the graphics of a {@code Elderly} using a {@code ElderlyCard}.
      */
-    class ElderlyListViewCell extends ListCell<Elderly> {
+    static class ElderlyListViewCell extends ListCell<Elderly> {
```

Comment:
Yup I think it's better if it's static. Could you change `VolunteerListViewCell` and `PairListViewCell` to static as well?